deeplearning4j-nn/src/main/java/org/deeplearning4j/datasets/iterator/callbacks/DefaultCallback.java

```diff
@@ -0,0 +1,54 @@
+package org.deeplearning4j.datasets.iterator.callbacks;
+
+import org.nd4j.linalg.api.concurrency.AffinityManager;
+import org.nd4j.linalg.dataset.api.DataSet;
+import org.nd4j.linalg.dataset.api.MultiDataSet;
+import org.nd4j.linalg.factory.Nd4j;
+
+/**
+ * @author raver119@gmail.com
+ */
+public class DefaultCallback implements DataSetCallback {
```

Comment:
I'm being a bit pendantic here: but "DefaultCallback" as a name doesn't really tell people what it does or if/why it should be used. Maybe something like "EnforceAfinityDataSetCallback" might be more informative?